src/adapter.js

```diff
@@ -695,7 +695,13 @@ export function parse({minim, source, generateSourceMap}, done) {
 
             // Responses can have bodies
             if (responseBody !== undefined) {
-              const bodyAsset = new Asset(JSON.stringify(responseBody, null, 2));
+              let formattedResponseBody = responseBody;
+
+              if (typeof(responseBody) === 'object') {
```

Comment:
Does this work? Can't valid JSON be any primitive value, from string, to array, to object? I will need to look at the Swagger spec, but I'm thinking we'll need to stringify anything other than a string (assuming a string is either a primitive string or JSON-stringified).